content/en/docs/setup/install/istioctl/index.md

```diff
@@ -45,14 +45,16 @@ The simplest option is to install the `default` Istio
 using the following command:
 
 {{< text bash >}}
-$ istioctl install
+$ istioctl install --manifests=manifests/
 {{< /text >}}
 
 This command installs the `default` profile on the cluster defined by your
 Kubernetes configuration. The `default` profile is a good starting point
 for establishing a production environment, unlike the larger `demo` profile that
 is intended for evaluating a broad set of Istio features.
 
+The `manifests` flag determines which installation manifests should be used for installation. The installation manifests are included in the release tar in the `manifests` directory.
```

Comment:
we should mention something about version compatibility early on in the doc e.g. istioctl is guaranteed to be compatible with the same minor version of manifests e.g. istioctl v1.7.2 is compatible with all 1.7.x manifests.